pennylane/devices/qutrit_mixed/measure.py

```diff
@@ -39,31 +39,28 @@
 )
 from .apply_operation import apply_operation
 
-alphabet_array = math.asarray(list(alphabet))
 
+def _map_indices_apply_operation(**kwargs):
+    """Map indices to wires.
 
-def _map_indices_apply_operation(
-    state_indices, row_indices, new_row_indices, **kwargs
-):  # pylint: disable=unused-argument
-    """Map indices to wires
     Args:
-        state_indices (str): Indices that are summed.
-        row_indices (str): Indices that must be replaced with sums.
-        new_row_indices (str): Tensor indices of the state.
-        **kwargs (dict): Stores indices calculated in `get_einsum_mapping`.
+        **kwargs (dict): Stores indices calculated in `get_einsum_mapping`:
```

Comment:
Thanks for updating this - as long as the other reviewers are okay with the convention here, this is fine with me.